_posts/2024-02-26-cache-refresh.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "A cache refresh for Trino"
+author: Manfred Moser
+image: /assets/blog/trino-cache-refresh.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Leon Bambrick once said "There are only two hard things in computer science:
+cache invalidation, naming things, and off-by-one errors" improving the initial
+wisdom from [Phil Karlton](https://www.karlton.org/2017/12/naming-things-hard/).
```

Comment:
I'm a little uncertain about this quote and citing two different individuals, as well as linking to this kinda sketchy website. I'm not familiar with the person we're referencing, and I don't think there's an expectation for them to be well-known names, so proper writing would need to introduce or explain background on them.

But something like, "There's a famous proverb: 'There are only two hard things...'" without attribution is actually fair game, as the quote really is a proverb at this point and [thus doesn't need to be attributed](https://style.mla.org/citing-famous-sayings/).

The Leon Bambrick variation isn't as much a proverb, and I'd think we can just drop the "off by one errors" portion of the quote to avoid the messiness of attributing it to a not-super-notable individual and needing to explain who they are.

Reply:
The website is not sketchy at all.. just old. And the two authors are both famous and I think we want to at least attribute to the Karlton site.. and it has lots more links to related stuff. I chose the Bambrick quote because its funny..  so it makes for a light entry to the blog post